core-blocks/gallery/block.js

```diff
@@ -236,6 +247,7 @@ class GalleryBlock extends Component {
 					</PanelBody>
 				</InspectorControls>
 				<ul className={ `${ className } align${ align } columns-${ columns } ${ imageCrop ? 'is-cropped' : '' }` }>
+					{ noticesUI }
```

Comment:
Can this be a child of `ul`?

Reply:
It should be outside of the ul during the rebase I made a mistake it was corrected. Thank you!